pom.xml

```diff
@@ -73,13 +74,12 @@
     <dependency>
       <groupId>javax.servlet</groupId>
       <artifactId>javax.servlet-api</artifactId>
```

Comment:
I think this dep could simply be deleted—implied for any plugin.

Reply:
ah yup. I reckon I might be a bit scrupulous and always declare in a pom the dependencies really used :)